openslides/global_settings.py

```diff
@@ -71,7 +67,7 @@
     'django.middleware.locale.LocaleMiddleware',
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
-    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'openslides.users.anonymous.AuthenticationMiddleware',
```

Comment:
I like the name "anonymous" more, because this is an AuthenticationMiddleware that (in difference to is original) allows anonymous "authentication".